### DISCUSSION TITLE
如何查看索引占用的额外空间大小

### DISCUSSION THREAD

**Discussion by gitzhou-sin** [apache/doris#37867] [A - General / Q&A]
### 我想知道索引列占用了多少额外空间，可以查询到吗？
我创建了一个表并插入了一些数据，表大小及tablets数量如下：
![image](https://github.com/user-attachments/assets/b854dac2-f068-4472-b123-b1885fb1ba42)
为了加速查询使用了ROLLUP
```sql
ALTER TABLE td_home_ygwip DROP ROLLUP r_index_mdip;
```
ROLLUP 成功后`show data;`内容如下
![image](https://github.com/user-attachments/assets/f9bda1a5-afa0-4174-9470-1b89f5abc1d7)
我的问题有三个：
1. 为什么ROLLUP后会导致tablets数量翻倍了
2. tablets翻倍的情况下，`size`列大小几乎无变化，但是我想知道ROLLUP后增加了多少额外存储
3. 可以查询到其他二级索引占用的额外空间吗？(我的表用到了倒排索引)

**Comment by ixzc** [comment]
中文问题在ask.selectdb.com提个帖子
